rumqttc/src/lib.rs

```diff
@@ -562,8 +562,6 @@ impl MqttOptions {
     /// Set number of seconds after which client should ping the broker
     /// if there is no other data exchange
     pub fn set_keep_alive(&mut self, duration: Duration) -> &mut Self {
-        assert!(duration.as_secs() >= 5, "Keep alives should be >= 5 secs");
-
         self.keep_alive = duration;
         self
     }
```

Comment:
according to mqtt spec:
> The Keep Alive is a time interval measured in seconds. Expressed as a 16-bit word, it is the maximum time interval that is permitted to elapse between the point at which the Client finishes transmitting one Control Packet and the point it starts sending the next. It is the responsibility of the Client to ensure that the interval between Control Packets being sent does not exceed the Keep Alive value. In the absence of sending any other Control Packets, the Client MUST send a PINGREQ Packet [MQTT-3.1.2-23].

keep alive should be in seconds but because we take `Duration` as a argument, which means it can technically contain nanoseconds, so ideally we should take `usize` as argument.

But to not break API for now we can verify that duration doesn't contain nanoseconds inside this method.

wdyt? @yatinmaan